scripts/bmm_examples/Example2_Loaiza&Souza2018.R

```diff
@@ -74,86 +75,116 @@ levels(wmData$ageGroup) <- c("Young","Old")
 #' Using priors (see step 3), we will constrain the means of both von Mises distributions
 #' to zero. Additionally, we will use priors to fix the precision (kappa) of the
 #' second von Mises to be zero (at least practically zero). 
-LS_mixFormula_bmm <- bf(
-  # Initializing the dependent variable
-  dev_rad ~ 1,
-  
+LS_formula <- bmf(
   # estimating fixed intercept & random intercept for kappa of the first von Mises
-  kappa ~ 0 + ageGroup:RI:cueCond + (0 + RI:cueCond || gr(id, by = ageGroup)), 
+  kappa ~ 0 + ageGroup + ageGroup:RI + ageGroup:cueCond + ageGroup:RI:cueCond  + 
+    (0 + RI + cueCond + RI:cueCond || gr(id, by = ageGroup)), 
   
   # estimating fixed intercept & random intercept for the mixing proportion 
   # of the target vonMises (i.e., p_mem)
-  thetat ~ 0 + ageGroup:RI:cueCond + (0 + RI:cueCond || gr(id, by = ageGroup))
+  thetat ~ 0 + ageGroup + ageGroup:RI + ageGroup:cueCond + ageGroup:RI:cueCond  + 
+    (0 + RI + cueCond + RI:cueCond || gr(id, by = ageGroup))
 )
 
-# constrain parameters using priors
-mixPriors <- prior(normal(0,0.5), class = "b", dpar = "theta1") +
-  prior(normal(0,0.5), class = "b", dpar = "kappa1")
+# check the default priors
+default_prior(LS_formula, data = wmData, model = LS_model)
+
+prior <- 
 
 # fit mixture model if there is not already a results file stored
-if (!file.exists(here("output","fit_E2_LS2018.RData"))) {
-  # fit the mixture model using brms
-  fit_LS2018_mixModel <- fit_model(
-    # include model information
-    formula = LS_mixFormula_bmm, # specify formula for mixture model
-    data    = wmData, # specify data used to estimate the mixture model
-    model_type = "2p",
-    prior   = mixPriors, # use the used defined priors,
-    
-    # save settings
-    sample_prior = TRUE,
-    save_pars = save_pars(all = TRUE),
-    
-    # add brms settings
-    warmup = warmup_samples,
-    iter = warmup_samples + postwarmup_samples, 
-    chains = nChains,
-    
-    # control commands for the sampler
-    control = list(adapt_delta = adapt_delta, 
-                   max_treedepth = max_treedepth)
-  )
+LS2018_fit <- bmm(
+  # include model information
+  formula = LS_formula, # specify formula for mixture model
+  data    = wmData, # specify data used to estimate the mixture model
+  model = LS_model,
   
-  # save results into file
-  save(fit_LS2018_mixModel, 
-       file = here("output","fit_E2_LS2018.RData"),
-       compress = "xz")
+  # save settings
+  sample_prior = TRUE,
+  save_pars = save_pars(all = TRUE),
   
-} else {
-  # load results file
-  load(file = here("output","fit_E2_LS2018.RData"))
-}
+  # add brms settings
+  warmup = warmup_samples,
+  iter = warmup_samples + postwarmup_samples, 
+  chains = nChains,
+  cores = parallel::detectCores(),
+  
+  # control commands for the sampler
+  control = list(adapt_delta = adapt_delta, 
+                 max_treedepth = max_treedepth),
+  
+  # save the results
+  file = here("output","fit_E2_LS2018_bmm")
```

Comment:
ok this one works! so it must be just a problem with the other script using an older bmm version?